sdcm/db_stats.py

```diff
@@ -792,3 +801,65 @@ def get_doc_data(self, key) -> Optional[dict]:
 
     def get_test_start_time(self):
         return self._stats['test_details'].get('start_time', None)
+
+
+def generate_node_capacity_query_postfix(node):
+    return f'{{mountpoint="{SCYLLA_DIR}", instance=~".*?{node.private_ip_address}.*?"}}'
+
+
+def get_used_capacity_gb(prometheus_db, node):  # pylint: disable=too-many-locals
```

Comment:
Why those aren't part of the PrometheusDBStats, like all the other ones defined in this file ?

Reply:
fixed